frontend/Dockerfile

```diff
@@ -1,13 +1,12 @@
 # multistage build of nginx image
 
 # stage 1 of 2
-FROM node:20-bullseye AS builder
+FROM node:22-alpine AS builder
 WORKDIR /app
 
 # install npm packages -- do this before copying everything else
 # to enable more efficient caching by docker (we don't reinstall packages when
 # we've only changed application code)
-RUN apt-get update && apt-get install -y build-essential python3 python-is-python3
```

Comment:
As far as I can tell, the frontend build doesn't use Python. This might be a holdover from building the tile servers.

Reply:
Removing this also allows us to build from Node Alpine, so smaller images.